### PR TITLE
extensions for mysql and postgresql added

### PR DESCRIPTION
**Pull request by mkarg75**
Depends-On: 391

I've added the required bits and pieces for mysql and postgresql while leaving mssql intact.

### CODE REVIEW

File: snafu/hammerdb/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
 # Download and install the hammer suite
-RUN curl -L 'https://downloads.sourceforge.net/project/hammerdb/HammerDB/HammerDB-3.2/HammerDB-3.2-Linux-x86-64-Install?r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fhammerdb%2Ffiles%2FHammerDB%2FHammerDB-3.2%2FHammerDB-3.2-Linux-x86-64-Install%2Fdownload&ts=1564587940&use_mirror=autoselect' -o hammer_installer
+RUN curl -L 'https://github.com/TPC-Council/HammerDB/releases/download/v3.3/HammerDB-3.3-Linux-x86-64-Install' -o hammer_installer
 RUN mkdir /hammer
 RUN chmod 755 hammer_installer
 RUN ./hammer_installer --prefix /hammer --mode silent
-COPY snafu/hammerdb/uid_entrypoint /usr/local/bin/
+COPY src/hammerdb/uid_entrypoint /usr/local/bin/
```
Comment (rsevilla87) [inline snafu/hammerdb/Dockerfile:22]: This should be left as is now

File: snafu/hammerdb/hammerdb_wrapper.py [hunk 2]
```diff
@@ -39,22 +72,76 @@ def __init__(self, parser):
...
+        if "db_postgresql_defaultdbase" in os.environ:
+            self.args.db_postgresql_defaultdbase = os.environ["db_postgresql_defaultdbase"]
+        if "db_postgresql_vacuum" in os.environ:
+            self.args.db_postgresql_vacuum = os.environ["db_postgresql_vacuum"]
+        if "db_postgresql_dritasnap" in os.environ:
+            self.args.db_postgresql_dritasnap = os.environ["db_postgresql_dritasnap"]
+        if "db_postgresql_oracompat" in os.environ:
+            self.args.db_postgresql_oracompat = os.environ["db_postgresql_oracompat"]
+        if "db_postgresql_storedprocs" in os.environ:
+            self.args.db_postgresql_storedprocs = os.environ["db_postgresql_storedprocs"]
```
Comment (rsevilla87) [inline snafu/hammerdb/hammerdb_wrapper.py:144]: The logic from the line 14 to this line can be reduced to:

```suggestion
        self.args.es_server = os.getenv("es_server", "")
        self.args.es_port = os.getenv("es_port", "")
        # generic benchmark parameters
        self.args.db_type = os.getenv("db_type", "")
        self.args.db_server = os.getenv("db_server", "")
        self.args.db_port = os.getenv("db_port", "")
        self.args.db_warehouses = os.getenv("db_warehouses", "")
        self.args.db_num_workers = os.getenv("db_num_workers", "")
        self.args.db_user = os.getenv("db_user", "")
        self.args.transactions = os.getenv("transactions", "")
        self.args.raiseerror = os.getenv("raiseerror", "")
        self.args.keyandthink = os.getenv("keyandthink", "")
        self.args.driver = os.getenv("driver", "")
        self.args.rampup = os.getenv("rampup", "")
        self.args.runtime = os.getenv("runtime", "")
        self.args.allwarehouse = os.getenv("allwarehouse", "")
        self.args.timeprofile = os.getenv("timeprofile", "")
        self.args.async_scale = os.getenv("async_scale", "")
        self.args.async_client = os.getenv("async_client", "")
        self.args.async_verbose = os.getenv("async_verbose", "")
        self.args.async_delay = os.getenv("async_delay", "")
        self.args.samples = os.getenv("samples", "")
        # database specific parameters
        # mssql
        self.args.db_mssql_tcp = os.getenv("db_mssql_tcp", "")
        self.args.db_mssql_azure = os.getenv("db_mssql_azure", "")
        self.args.db_mssql_authentication = os.getenv("db_mssql_authentication", "")
        self.args.db_mssql_linux_authent = os.getenv("db_mssql_linux_authent", "")
        self.args.db_mssql_odbc_driver = os.getenv("db_mssql_odbc_driver", "")
        self.args.db_mssql_linux_odbc = os.getenv("db_mssql_linux_odbc", "")
        self.args.db_mssql_imdb = os.getenv("db_mssql_imdb", "")
        self.args.db_mssql_bucket = os.getenv("db_mssql_bucket", "")
        self.args.db_mssql_durability = os.getenv("db_mssql_durability", "")
        self.args.db_mssql_checkpoint = os.getenv("db_mssql_checkpoint", "")
        # mysql 
        self.args.db_mysql_storage_engine = os.getenv("db_mysql_storage_engine", "")
        self.args.db_mysql_partition = os.getenv("db_mysql_partition", "")
        # postgresql
        self.args.db_postgresql_superuser = os.getenv("db_postgresql_superuser", "")
        self.args.db_postgresql_superuser_pass = os.getenv("db_postgresql_superuser_pass", "")
        self.args.db_postgresql_defaultdbase = os.getenv("db_postgresql_defaultdbase", "")
        self.args.db_postgresql_vacuum = os.getenv("db_postgresql_vacuum", "")
        self.args.db_postgresql_dritasnap = os.getenv("db_postgresql_dritasnap", "")
        self.args.db_postgresql_oracompat = os.getenv("db_postgresql_oracompat", "")
        self.args.db_postgresql_storedprocs = os.getenv("db_postgresql_storedprocs", "")

```

File: snafu/hammerdb/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,8 @@
 FROM registry.access.redhat.com/ubi8:latest
 
 # install requirements
+#COPY image_resources/centos8.repo /etc/yum.repos.d/centos8.repo
+#COPY image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
```
Comment (rsevilla87) [inline snafu/hammerdb/Dockerfile:5]: Why this?

**Review by rsevilla87 — APPROVED** [review]
LGTM!

File: snafu/hammerdb/trigger_hammerdb.py [hunk 4]
```diff
@@ -41,89 +100,115 @@ def _parse_stdout(self, stdout):
...
+                print("+{}+".format("-" * (115)))
+                i += 1
+            current_worker *= 2
 
     def emit_actions(self):
         timestamp = str(int(time.time()))
+        logger.info("Starting hammerdb run")
         stdout = self._run_hammerdb()
-        # stdout = _fake_run()
+        # stdout = self._fake_run()
```
Comment (dry923) [inline snafu/hammerdb/trigger_hammerdb.py:197]: remove comment

File: snafu/hammerdb/Dockerfile (hunk 3 above)
Comment (dry923) [reply snafu/hammerdb/Dockerfile:5]: @mkarg75 clean up comments

**Pull request merged**